backend/src/main/scala/cromwell/backend/async/KnownJobFailureException.scala

```diff
@@ -8,8 +8,9 @@ abstract class KnownJobFailureException extends Exception {
   def stderrPath: Option[Path]
 }
 
-final case class WrongReturnCode(jobTag: String, returnCode: Int, stderrPath: Option[Path]) extends KnownJobFailureException {
-  override def getMessage = s"Job $jobTag exited with return code $returnCode which has not been declared as a valid return code. See 'continueOnReturnCode' runtime attribute for more details."
+final case class WrongReturnCode(jobTag: String, returnCode: Int, stderrPath: Option[Path], errorMessage: String = "") extends KnownJobFailureException {
```

Comment:
Yes, I have, but I ended up with this solution. Are you suggesting to use Option with default value None or to use Option without default value?
The problem is that I need to add an additional field to this class. For backward compatibility,  I've used the default field here. I thought that I can use something like
`errorMessage: Option[String] = None`
But it looked overengineered for me. But if this is more correct, I will make changes.

Reply:
As for me it's better to use Option with default value None.

Reply:
I think Option is not overengineering since it would be easier to see when `errorMessage` is here or not. Check `getMessage` method below. With just empty string it's gonna look confusing , but as an Option it'll give you a value of `None`.